app/views/compute_resources_vms/form/openstack/_base.html.erb

```diff
@@ -18,3 +18,10 @@
 <%= selectable_f f, :network, compute_resource.address_pools, { include_blank: 'None' }, { :label => _("Floating IP network"), :label_size => "col-md-2" } %>
 <%= checkbox_f f, :boot_from_volume, {:label => _("Boot from volume"), :label_size => "col-md-2", :help_inline => _("Create new boot volume from image")}, "true", "false" %>
 <%= text_f f, :size_gb, { :label => _("New boot volume size (GB)"), :label_size => "col-md-2", :help_inline => _("Defaults to image size if left blank") } %>
+<%= selectable_f f, :scheduler_hint_filter, compute_resource.possible_scheduler_hints, { include_blank: 'None' }, { :label => _("Scheduler hint filter"), :label_size => "col-md-2", :'data-url' => scheduler_hint_selected_hosts_path,
+                                                                                                              :onchange => 'schedulerHintFilterSelected(this);' } %>
+<div id="scheduler_hint_wrapper">
+  <%= if @host && @host.compute_attributes[:scheduler_hint_filter]
```

Comment:
1. The `if` is to force rendering the proper partial in case the form was submitted but returned error due to validation for example.
2. Not sure if it applies only to new hosts, I think so, but will amend it in the proper partial